doc/examples/tracking_tissue_classifier.py

```diff
@@ -0,0 +1,319 @@
+                                                     threshold_classifier,
+                                                     seeds,
+                                                     affine,
+                                                     step_size=.5,
+                                                     return_all=True)
+
+save_trk("deterministic_threshold_classifier_all.trk",
+         all_streamlines_threshold_classifier,
+         affine,
+         labels.shape)
+
+streamlines = [sl for sl in all_streamlines_threshold_classifier]
+
+fvtk.clear(ren)
+fvtk.add(ren, fvtk.line(streamlines, line_colors(streamlines)))
+fvtk.record(ren, out_path='all_streamlines_threshold_classifier.png',
+            size=(600, 600))
+
+"""
+.. figure:: all_streamlines_threshold_classifier.png
+ :align: center
+
+ **Deterministic tractography using a thresholded fractional anisotropy map.**
+"""
+
+
+"""
+Binary Tissue Classifier
+------------------------
+A binary mask can be used as tissue classifier for the tractography.
+Here, we show how to obtain the binary tissue classifier from
+the white matter mask defined above. The binary tissue classifier uses a
+nearest-neighbourhood interpolation method at the tracking position.
+
+**Parameters**
+
+- mask: numpy array [:, :, :]
+
+**Stopping criterion**
+
+- 'ENDPOINT': mask = 0
+- 'OUTSIDEIMAGE': tracking point outside of mask
+- 'TRACKPOINT': stopped because no direction are available
+- 'INVALIDPOINT': N/A
+"""
+
+from dipy.tracking.local import BinaryTissueClassifier
+
+binary_classifier = BinaryTissueClassifier(white_matter)
+
+fig = plt.figure()
+plt.xticks([])
+plt.yticks([])
+fig.tight_layout()
+plt.imshow(white_matter[:, :, data.shape[2] / 2].T, cmap='gray', origin='lower',
+           interpolation='nearest')
+fig.savefig('white_matter_mask.png')
+
+"""
+.. figure:: white_matter_mask.png
+ :align: center
+
+ **White matter binary mask.**
+"""
+
+all_streamlines_binary_classifier = LocalTracking(dg,
+                                                  binary_classifier,
+                                                  seeds,
+                                                  affine,
+                                                  step_size=.5,
+                                                  return_all=True)
+
+save_trk("deterministic_binary_classifier_all.trk",
+         all_streamlines_binary_classifier,
+         affine,
+         labels.shape)
+
+streamlines = [sl for sl in all_streamlines_binary_classifier]
+fvtk.clear(ren)
+fvtk.add(ren, fvtk.line(streamlines, line_colors(streamlines)))
+fvtk.record(ren, out_path='all_streamlines_binary_classifier.png',
+            size=(600, 600))
+
+"""
+.. figure:: all_streamlines_binary_classifier.png
+ :align: center
+
+ **Deterministic tractography using a binary white matter mask.**
+"""
+
+"""
+ACT Tissue Classifier
+---------------------
+Anatomically-constrained tractography (ACT) [Smith2012]_ uses information from
```

Comment:
I didn't include the method of my paper (CMC) in this PR. I will included it when PFT gets in.